packages/adapter-components/src/fetch/element/openAPI/types.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ *                      Copyright 2024 Salto Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import _ from 'lodash'
+import { ObjectType, TypeMap } from '@salto-io/adapter-api'
+import { pathNaclCase } from '@salto-io/adapter-utils'
+import { logger } from '@salto-io/logging'
+import { getParsedSchemas } from './parser'
+import { OpenAPIDefinition } from '../../../definitions/system/sources'
+import { defineAdditionalTypes } from '../../../elements/swagger/type_elements/type_config_override'
+import { typeAdder } from '../../../elements/swagger/type_elements/element_generator'
+import { ElementAndResourceDefFinder } from '../../../definitions/system/fetch/types'
+import { FetchApiDefinitionsOptions } from '../../../definitions/system/fetch'
+import { TYPES_PATH, SUBTYPES_PATH } from '../../../elements/constants'
+
+const log = logger(module)
+
+/**
+ * openAPI types are created by under the subtypes folder, modify path of top level instances to be under the types folder
```

Comment:
```suggestion
 * openAPI types are created under the subtypes folder, modify path of top level instances to be under the types folder
```
(or did you mean "by default"?)